src/plugins/chart_expressions/expression_xy/common/expression_functions/extended_annotation_layer.ts

```diff
@@ -30,7 +30,11 @@ export function extendedAnnotationLayerFunction(): ExpressionFunctionDefinition<
         help: strings.getAnnotationLayerSimpleViewHelp(),
       },
       annotations: {
-        types: ['manual_point_event_annotation', 'manual_range_event_annotation'],
+        types: [
```

Comment:
There's also `simplifiedView` param for the Lens layer. `layerId` is also included for consistency, in case if it's needed in more usecases in the future, but yeah, the good usecase you mentioned is `inspector`.